Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/LayerIntegrationTest.kt

```diff
@@ -596,8 +596,8 @@ class LayerIntegrationTest {
         LayerMenuViewInteraction.onLayerMenuView()
             .performOpen()
 
-        onView(withId(R.id.pocketpaint_layer_side_nav_button_visibility)).check(ViewAssertions.matches(Matchers.not(isClickable())))
-        onView(withId(R.id.pocketpaint_layer_side_nav_button_opacity)).check(ViewAssertions.matches(Matchers.not(isClickable())))
+        onView(withId(R.id.pocketpaint_layer_side_nav_button_visibility)).check(ViewAssertions.matches(Matchers.not(isEnabled())))
+        onView(withId(R.id.pocketpaint_layer_side_nav_button_opacity)).check(ViewAssertions.matches(Matchers.not(isEnabled())))
     }
```

Comment:
Lines are bit too long.
`onView(withId(R.id.pocketpaint_layer_side_nav_button_visibility))
            .check(ViewAssertions.matches(Matchers.not(isEnabled())))
onView(withId(R.id.pocketpaint_layer_side_nav_button_opacity))
            .check(ViewAssertions.matches(Matchers.not(isEnabled())))`

Reply:
removed the redundant ViewAssertions and made lines shorter